test/edu-benefits/components/ReviewCollapsiblePanel.unit.spec.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import ReviewCollapsiblePanel from '../../../src/js/edu-benefits/components/ReviewCollapsiblePanel';
+import { createVeteran } from '../../../src/js/edu-benefits/utils/veteran';
+
+describe('<ReviewCollapsiblePanel>', () => {
+  function FakeFields() {}
+  function FakeReview() {}
+  it('should render fields to update', () => {
```

Comment:
Since they're all pretty similar, you could wrap these tests in a function and just tweak the parameters for the things you want to test against.

Check out https://github.com/department-of-veterans-affairs/vets-website/blob/master/test/common/Nav/Nav.unit.spec.jsx#L89-L105 for an example